src/internal/components/cartesian-chart/bottom-labels.tsx

```diff
@@ -82,7 +82,6 @@ function BottomLabels({
     <g
       transform={`translate(0,${height})`}
       className={styles['labels-bottom']}
-      focusable={false}
```

Comment:
I can't find this attribute on the spec https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g . Looking for input here, should this be `tabIndex={-1}`?